### PR TITLE
[ACR-1759] Add mv_source_h timestamp API doc

### PR DESCRIPTION
**Pull request by KwangSon**
### Change Description ###
Add description about timestamp data API for mv_source_h
 - [ACR-1759](https://code.sec.samsung.net/jira/browse/ACR-1759)

### CODE REVIEW

File: docs/application/native/guides/multimedia/image-classification.md [hunk 1]
```diff
@@ -194,6 +194,27 @@ To classify an image, follow these steps:
     if (error_code != MEDIA_VISION_ERROR_NONE)
         dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
     ```
+## Timestamp
+
+Time is important for camera data, and you may want to bring time information from the camera system to mediavision.  
+In such a case, if you want to include time information in the mv_source data, there is a method as follows.
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-classification.md:200]:
```suggestion
In such a case, if you want to include time information in the mv_source data, there is a method.
```

File: docs/application/native/guides/multimedia/image-classification.md [hunk 2]
```diff
@@ -194,6 +194,27 @@ To classify an image, follow these steps:
     if (error_code != MEDIA_VISION_ERROR_NONE)
         dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
     ```
+## Timestamp
+
+Time is important for camera data, and you may want to bring time information from the camera system to mediavision.  
+In such a case, if you want to include time information in the mv_source data, there is a method as follows.  
+Use `mv_source_{get,set}_timestamp()` to handle timedata.
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-classification.md:201]:
```suggestion
Use `mv_source_{get,set}_timestamp()` to handle timedata:
```

**Review by omar-srbd — COMMENTED** [review]
Hello @KwangSon, I have reviewed the file in this PR and left some suggestions.
Please check and incorporate them accordingly.
Thank you.

File: docs/application/native/guides/multimedia/image-classification.md [hunk 3]
```diff
@@ -194,6 +194,27 @@ To classify an image, follow these steps:
...
+        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+    error_code = mv_source_set_timestamp(mv_source, 2);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+    error_code = mv_source_get_timestamp(mv_source, &timestamp);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+    ```
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-classification.md:217]:
```suggestion
```c
mv_source_h mv_source;
uint64_t timestamp;
error_code = mv_create_source(&mv_source);
if (error_code != MEDIA_VISION_ERROR_NONE)
    dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);

error_code = mv_source_set_timestamp(mv_source, 2);
if (error_code != MEDIA_VISION_ERROR_NONE)
    dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);

error_code = mv_source_get_timestamp(mv_source, &timestamp);
if (error_code != MEDIA_VISION_ERROR_NONE)
    dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
    ```
```

**Review by omar-srbd — COMMENTED** [review]
Please fix the alignment of the code snippet.
Thank you.

File: docs/application/native/guides/multimedia/image-classification.md (hunk 3 above)
Comment (KwangSon) [reply docs/application/native/guides/multimedia/image-classification.md:217]: line 217 should be without indent right?

Comment (omar-srbd) [reply docs/application/native/guides/multimedia/image-classification.md]: @KwangSon, yes. The indentation is proper now.

**Review by omar-srbd — APPROVED** [review]
@KwangSon, thanks for making all the changes.

@asm-foysal, I am approving this PR for merging.

**Comment by KwangSon** [comment]
@asm-foysal @omar-srbd Could you merge and process ACR?

**Pull request merged**